qubesadmin/tools/qvm_ls.py

```diff
@@ -107,6 +107,8 @@ def format(self, vm):
                 ret = vm
                 for attrseg in self._attr.split('.'):
                     ret = getattr(ret, attrseg)
+                    if hasattr(ret, '__iter__') and not isinstance(ret, str):
+                        ret = ' '.join(map(str, ret))
```

Comment:
better separate with coma (which also cannot be part of the tag name), otherwise columns may be ambiguous.

Reply:
I meant just coma, to avoid spaces within the column content...

Reply:
sorry about that, I just amended the previous commit to replace space with a comma